app/stac_api/models.py

```diff
@@ -229,13 +664,10 @@ def update_bbox_extent(self, action, item_geom, item_id, item_name):
         collection foreign key. If there is no spatial bbox yet, the one of the geometry of the
         item is being used.
         '''
-
         try:
             # insert (as item_id is None)
             if action == 'insert':
-                logger.debug('Updating collections extent_geometry'
-                             ' as a item has been inserted'
-                             )
+                logger.debug('Updating collections extent_geometry' ' as a item has been inserted')
```

Comment:
It might be a good idea to add the `collection_name` as an `extra` argument to all log entries where available (as either `collection_name` or simply `collection`) this will allow to easily filter message belonging to a certain collection (that might be causing troubles)

Reply:
done